dashboard/views.py

```diff
@@ -5,9 +5,11 @@
 from datetime import timedelta
 from json import JSONDecodeError
 
+from constance import config
 import jsonschema
 import numpy as np
 import pandas as pd
+
 from django.conf import settings
```

Comment:
Nit-pick: The new seems unnecessary. I don't remember if I set this pattern up or not, but I guess I'd stick with the separate alphabetical order for direct and `from` imports.

```suggestion
import jsonschema
import numpy as np
import pandas as pd
from constance import config
from django.conf import settings
```